docker-unit-tests.sh

```diff
@@ -1,4 +1,11 @@
-#!/bin/bash
+#!/bin/bash -x
+
+BUILD_CORES=$(echo "scale=0 ; `nproc` / 1.337" | bc)
+
+if [[ "$GITHUB_REPOSITORY" == "" ]]; then
+  #Default
+  BUILD_CORES=8
+fi
 
 echo "Mounting $(pwd)/io in ubuntu and running unit tests"
-docker run --rm -i -v $(pwd):/io ubuntu sh -c '/io/release-build/xahaud -u'
+docker run --rm -i -v $(pwd):/io -e BUILD_CORES=$BUILD_CORES ubuntu sh -c '/io/release-build/xahaud --unittest-jobs $BUILD_CORES -u'
```

Comment:
-e BUILD_CORES=$BUILD_CORES 

Is that actually doing anything
Rippled doesn't honor that in anyway afaik?

Reply:
`/io/release-build/xahaud --unittest-jobs $BUILD_CORES -u` is executed in the container, but the environment variable BUILD_CORES is not set in the container.
Set the container environment variables with the -e option.

In fact, if the -e option is not used, the workflow fails.
https://github.com/Xahau/xahaud/pull/483/commits/1ef590e975190683bb5a0fd1507fd232f3f28f8d

Reply:
Thanks for explanation. 

So $BUILD_CORES is not evaluated until inside the container due to single quotes